### PR TITLE
Update events.json

### PR DESCRIPTION
**Pull request by ananya**
Issue Number #70 
Changes proposed in this pull request : added more events
[Live Link](provide link here)

### CODE REVIEW

File: static/js/events.json [hunk 1]
```diff
@@ -269,6 +269,62 @@
 			"date": "2017-02-01",
 			"location": "Taipei, Taiwan",
 			"link": "https://www.meetup.com/Women-Who-Code-Taipei/events/246714485/"
+		},
+
```
Comment (gauthamzz) [inline static/js/events.json]: where is hack in the north
where is opencode 😄

**Comment by Anupam-dagar** [comment]
@ananya-agrawal resolve merge conflicts

**Comment by ananya** [comment]
@Anupam-dagar image was added for contribution section.

**Review by gauthamzz — CHANGES_REQUESTED** [review]
Didn’t I already merge this

**Comment by ananya** [comment]
@gauthamzz there were two pr containing different events . HINT was added in the earlier one and added opencode in this. Maybe while fixing conflicts two events got repeated so i will verify that after this pr gets merged . Tell if some changes are to be made.

File: static/js/events.json [hunk 2]
```diff
@@ -271,7 +271,57 @@
...
+		},
+		
+		{
+			"title": "Mathematics of Machine Learning",
+			"date": "2018-01-07",
+			"location": "Delhi,India",
+			"link": "https://www.meetup.com/Women-Who-Code-Delhi/events/246927838/"
+    },
+    {
+      "title": "Hack-36",
```
Comment (Anupam-dagar) [inline static/js/events.json:324]: proper indentation needed, everything else LGTM :+1:

**Review by Anupam-dagar — COMMENTED** [review]
indentation needed

**Pull request closed without merging**